lightning/src/ln/channel.rs

```diff
@@ -3328,14 +3330,7 @@ impl<Signer: Sign> Channel<Signer> {
 
 	/// Allowed in any state (including after shutdown)
 	pub fn get_announced_htlc_max_msat(&self) -> u64 {
-		return cmp::min(
-			// Upper bound by capacity. We make it a bit less than full capacity to prevent attempts
-			// to use full capacity. This is an effort to reduce routing failures, because in many cases
-			// channel might have been used to route very small values (either by honest users or as DoS).
-			self.channel_value_satoshis * 1000 * 9 / 10,
-
-			Channel::<Signer>::get_holder_max_htlc_value_in_flight_msat(self.channel_value_satoshis)
-		);
+		self.get_holder_max_htlc_value_in_flight_msat()
```

Comment:
Can we deprecate this method and instead make `get_holder_max_htlc_value_in_flight_msat` a pub fn ?